release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -24,7 +24,14 @@ submodule openconfig-bgp-common-multiprotocol {
     for multiple protocols in BGP. The groupings are common across
     multiple contexts.";
 
-  oc-ext:openconfig-version "6.0.0";
+  oc-ext:openconfig-version "8.0.0";
```

Comment:
note: there is a 7.0.0 update which has been accepted previously but not published to public yet.

This can now become a minor revision since if comment below on prefix-limit is accepted, there is no breaking change.

Reply:
Keeping the version at 8.0.0 for current alignment